x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/Literal.java

```diff
@@ -32,6 +32,14 @@ public Literal(Source source, Object value, DataType dataType) {
         this.value = value;
     }
 
+    public static Literal trueWithSource(Source source) {
+        return new Literal(source, Boolean.TRUE, DataTypes.BOOLEAN);
+    }
+
+    public static Literal falseWithSource(Source source) {
+        return new Literal(source, Boolean.FALSE, DataTypes.BOOLEAN);
+    }
+
```

Comment:
QL classes need to remain slim - these type of utility methods are more trouble than their worth.
There are already utilities that provide similar functionality - see my previous comments.